src/main/java/dev/miku/r2dbc/mysql/Binding.java

```diff
@@ -65,6 +78,26 @@ PreparedExecuteMessage toMessage(int statementId, boolean immediate) {
         return new PreparedExecuteMessage(statementId, immediate, values);
     }
 
+    Mono<String> toSql(List<String> sqlParts) {
```

Comment:
If it's not required, then return a `String`. Indirections via `Mono` and `Flux` come with a performance penalty which can be measured in simple code blocks like these.

Reply:
Yes, that's I want. But, `Blob` and `Clob` parameter was using `Publisher<T>`, so if we need support parameter `Blob` and `Clob` without any blocking, it should be `Mono<String>`.